### PR TITLE
HCX-9 Add Planado APIv2 connector

### PR DESCRIPTION
**Pull request by TimAle**
http://vavr.io/

### CODE REVIEW

File: src/org/camunda/latera/bss/connectors/PlanadoV2.groovy [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+import org.camunda.bpm.engine.delegate.DelegateExecution
+import org.camunda.latera.bss.http.HTTPRestProcessor
+import org.camunda.latera.bss.logging.SimpleLogger
+import static org.camunda.latera.bss.utils.Numeric.toIntSafe
+import static org.camunda.latera.bss.utils.StringUtil.joinNonEmpty
+import java.security.MessageDigest
+import io.vavr.control.Try
+
+class PlanadoV2 {
+  Integer version
```
Comment (vladdoncheg) [inline src/org/camunda/latera/bss/connectors/PlanadoV2.groovy]: You dont need it as I can see

File: src/org/camunda/latera/bss/connectors/PlanadoV2.groovy [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+  HTTPRestProcessor http
+  SimpleLogger logger
+
+  PlanadoV2(DelegateExecution execution) {
+    this.logger =  new SimpleLogger(execution)
```
Comment (sugarfree1) [inline src/org/camunda/latera/bss/connectors/PlanadoV2.groovy]: i would argue that we need two spaces

File: src/org/camunda/latera/bss/connectors/PlanadoV2.groovy [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+    def ENV     =  System.getenv()
+
+    String url   =  execution.getVariable('planadoUrl')                                             ?: ENV['PLANADO_URL']   ?: 'https://api.planadoapp.com'
```
Comment (sugarfree1) [inline src/org/camunda/latera/bss/connectors/PlanadoV2.groovy]: and here

**Comment by sugarfree1** [comment]
don't forget to update changelog

**Comment by dolfinus** [comment]
Please don't forget to backport to v1.4

**Pull request merged**